utilities/blob_db/blob_db.h

```diff
@@ -120,6 +109,8 @@ class BlobDB : public StackableDB {
     return PutWithTTL(options, DefaultColumnFamily(), key, value, ttl);
   }
 
+  // Put with expiration. Key with expiration time equal to -1
+  // means the key don't expire.
   virtual Status PutUntil(const WriteOptions& options,
                           ColumnFamilyHandle* column_family, const Slice& key,
                           const Slice& value, int32_t expiration) = 0;
```

Comment:
Since expiration is of int32_t here instead of unsigned, it looks like we can only support half of uint32_t::max. In all other places we use uint32_t for ttl and uint64_t for expiration.